chain/chain/src/test_utils.rs

```diff
@@ -44,18 +45,29 @@ use crate::types::{ApplyTransactionResult, BlockHeaderInfo, ChainGenesis};
 use crate::{BlockHeader, DoomslugThresholdMode, RuntimeAdapter};
 
 #[derive(
-    BorshSerialize, BorshDeserialize, Serialize, Hash, PartialEq, Eq, Ord, PartialOrd, Clone, Debug,
+    BorshSerialize,
```

Comment:
we don't need to add this to test-only structs

Reply:
We need it for KeyValueRuntime in order for it to implement RuntimeAdater trait.

If needed I can change the code, so we don't do it for structures used by KeyValueRuntime,